bin/reth/src/commands/import.rs

```diff
@@ -61,6 +61,10 @@ pub struct ImportCommand {
     )]
     chain: Arc<ChainSpec>,
 
+    /// Disables execution stage.
+    #[arg(long, value_name = "DISABLE_EXECUTION", verbatim_doc_comment)]
+    disable_execution: bool,
```

Comment:
don't think this needs a value name, it's just a flag

also, should add `default_value_t` so it defaults to false (I don't think it does this by default?)

Reply:
it's a boolean flag, passing it sets to true, otherwise it's false